doc/sphinx/proofs/writing-proofs/proof-mode.rst

```diff
@@ -261,6 +274,11 @@ When the proof is completed, you can exit proof mode with commands such as
 
    .. seealso:: :cmd:`Proof with`
 
+   .. todo PR: Add a description for Proof Mode?
+
+.. cmd:: Proof Mode @string
+   :undocumented:
```

Comment:
Yes, a bug.  BTW, you do realize that Ltac2 doesn't support `idtac`?  Other thoughts:
- Not sure changing `Proof Mode` in the middle of a proof is a good idea or that it's a great command to have
- Probably should have been `Set Proof Mode`

```
From Ltac2 Require Import Ltac2.

Goal True.
Fail idtac. (* Unbound value idtac *)
```

Reply:
> BTW, you do realize that Ltac2 doesn't support `idtac`?

That's on purpose. The Ltac2 doc says:

>idtac is no more. Either use () if you expect nothing to happen, (fun () => ()) if you want a thunk (see next section), or use printing primitives from the Message module if you want to display something.



>Not sure changing `Proof Mode` in the middle of a proof is a good idea or that it's a great command to have

It's probably not very much used (since it is not documented), but I think this is not completely absurd to do that.

>Probably should have been `Set Proof Mode`

But then it would not respect the default locality behavior of options. There are no options whose scope is limited to the current proof (although maybe they should).

>AFAIK there's no way to show the current proof mode

That's unfortunate, but we have `Test Default Proof Mode` to know the default proof mode and if you are in the middle of a proof, you can look at the proof to see if there are any uses of the `Proof Mode` command.